spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -185,6 +188,10 @@ public Filter[] pushFilters(Filter[] filters) {
     return postScanFilters.toArray(new Filter[0]);
   }
 
+  private boolean unPartitioned() {
```

Comment:
Sorry just noticed, can we make it say 'unpartitioned'?  It matches PartitionSpec.unpartitioned()

Reply:
Sure, updated.